src/main/java/seedu/duke/command/DeleteNoteCommand.java

```diff
@@ -48,22 +48,25 @@ public DeleteNoteCommand(String title) {
 
     @Override
     public String execute() {
+        boolean isDeleted;
         try {
             // If there is no title, delete note by index. Else delete by title.
             if (title.isBlank()) {
                 String deletedTitle = notebook.getNotes().get(index).getTitle();
                 notebook.deleteNote(index);
 
-                return Formatter.formatString(COMMAND_SUCCESSFUL_MESSAGE + deletedTitle, false);
+                return Formatter.formatString(COMMAND_SUCCESSFUL_MESSAGE + deletedTitle);
             } else {
-                notebook.deleteNote(title);
+                isDeleted = notebook.deleteNote(title);
 
-                return Formatter.formatString(COMMAND_SUCCESSFUL_MESSAGE + title, false);
+                if (isDeleted) {
+                    return Formatter.formatString(COMMAND_SUCCESSFUL_MESSAGE + title);
+                } else {
+                    return Formatter.formatString(COMMAND_UNSUCCESSFUL_MESSAGE);
+                }
             }
-        } catch (SystemException exception) {
-            return Formatter.formatString(exception.getMessage(), false);
-        }  catch (IndexOutOfBoundsException exception) {
-            return Formatter.formatString(COMMAND_UNSUCCESSFUL_MESSAGE, false);
+        } catch (IndexOutOfBoundsException exception) {
+            return Formatter.formatString(COMMAND_UNSUCCESSFUL_MESSAGE);
```

Comment:
Possible to let notebook.deleteNote(index) return a boolean as well. So that can move the isDeleted outside of the if else (titleIsBlank) and do a consolidated return..
i.e.

if (titleIsBlank) {
title = getTitle()
isDeleted = notebook.deleteNote(index)
} else {
isDeleted = notebook.deleteNote(title)
}

if isDeleted
return success else return unsuccessful

Reply:
deleteNote(index) does not return boolean, I made it return true. Also I cannot remove the exception, when it getNotes it will return an exception when the index is out of bounds.